docs-src/lib.rkt

```diff
@@ -50,6 +50,19 @@
 (define (jsin . contents)
   (apply mint #:inline? #t "javascript" contents))
 
+(define (urlin . contents)
```

Comment:
I think that this really just `@tt`, because I think you are not using `mint-define` to generate URL xrefs, but I'm not totally sure on that